### ISSUE TITLE
Bug and Enhancement

### ISSUE THREAD

**Issue by RXWatcher** [ricreis394/plex-library-html#5]
Bug:
1) The library list doesn't scroll on Chrome or Firefox. I have a plex instance with a lot of libraries(like 20) and I'm unable to scroll down to all of them. I'd think this should be a simple fix with just adding the ability to scroll there.

Enhancement:
2) I'd like the ability to either only include specific libraries or exclude specific libraries. plex-library-html will grab and display all of them which can be an issue if you have ones you dont want to present.

Thanks again for making this. It is the best one I've seen so far of any of the plex library presentation apps.

**Comment by plckr** [comment]
> 1. The library list doesn't scroll on Chrome or Firefox. I have a plex instance with a lot of libraries(like 20) and I'm unable to scroll down to all of them. I'd think this should be a simple fix with just adding the ability to scroll there.

I'll check it asap


> 2) I'd like the ability to either only include specific libraries or exclude specific libraries. plex-library-html will grab and display all of them which can be an issue if you have ones you dont want to present.

Never tried but you could create a user limited to libraries you want to display. And add that user's token into PHP settings file
Let me know if it works

**Comment by RXWatcher** [comment]
https://github.com/ricreis394/plex-library-html/commit/e19d635491fc3c641f363b6ad0a1e5e81f7c9c1f does fix the scroll. Thanks!

Using a token from a non-admin account will not filter the libraries. It won't load at all. If I put back in the admin token then it works again.

**Comment by plckr** [comment]
I meant add a "friend" to plex, limited to certain libraries. I guess you added a "domestic user"

let me know if it worked

**Comment by RXWatcher** [comment]
No, I tried with a user who is not an admin to the server and who is not a managed user. It's just a new plex account that was invited to the server to only 2 libraries.  I pulled it's token, added it to plex-library-html and restarted it. It will look like it can't connect. I just get a spinning icon on calling the URL. See below as an example of what you get. If I go to /api/server (found in another issue) I get a 404 error. This works properly if I go back and put the admin users token back in. I do not have debug logging enabled any longer to pull the logs on exactly whats happening.

![image](https://user-images.githubusercontent.com/14085001/147095591-dedd6062-b881-45dc-b7a1-3744c09f2f2d.png)
